public/js/app.js

```diff
@@ -0,0 +1,110 @@
+// Call function getNextMove(player, coord)
+// player = 'computer' or 'human'
+// coord is an element in [0,1,2,3,4,5,6,7,8]
+// The function returns an element in [0,1,2,3,4,5,6,7,8]
+// as the computer's next move, 'computer', 'human', or 'tie'
+
+computerMoves = []
+humanMoves = []
+board = []
+wins = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+
+function getNextMove(player, coord){
+    _updateBoard(player, coord);
+    var nextMove = _nextMove(_otherPlayer(player))[0];
+    _updateBoard(_otherPlayer(player), nextMove);
+    console.log(nextMove);
+}
+
+function _updateBoard(player, coord) {
+    board.push(coord);
+    if (player == 'computer')
+        computerMoves.push(coord);
+    else
+        humanMoves.push(coord);
+}
+
+function _undoBoard(player, coord) {
+    board.pop();
+    if (player == 'computer')
+        computerMoves.pop();
+    else
+        humanMoves.pop();
+}
+
```

Comment:
Hard coding names like 'computer' and 'human' is generally considered bad because if you have them all over the place, changing the name means you have to change them in all those spots you use them.  Better to place them in a variable, and refer to the variable throughout your program.  So something like app.player = 'player'.